### ISSUE TITLE
Timezone settings are inaccessible (no screenreader output)

### ISSUE THREAD

**Issue by elioqoshi** [CanonicalLtd/desktop-design#158]
(reported by @MarcoZehe)

## How to reproduce

1. With Orca on, open Settings.
2. Find the Search field, activate it with Space, and enter „date“.
3. Tab until you reach the time zone item and press Space.
4. Enter your city.
5. Press Tab.

## Result
You won’t hear anything. You cannot make a selection. Orca’s Flat Review also only sees the search field and the unlabeled Find and Clear icons in that window, nothing else

--- 

I am not sure whether this is a Ubuntu specific issue or also present in GNOME. Regardless, we should address it.
@kenvandine, do you know who would be the best person to help on this?

**Comment by elioqoshi** [comment]
Tobias from GNOME is addressing these in the GNOME issue tracker:
https://gitlab.gnome.org/Teams/Design/settings-mockups/-/issues/23

**Comment by elioqoshi** [comment]
I am having a look at this again next iteration. Talked with the GNOME design team to get this merged upstream as well.

**Comment by SirSamTumless** [comment]
Can I close this @elioqoshi ?

**Comment by elioqoshi** [comment]
Nope! Still relevant.

**Comment by SirSamTumless** [comment]
Are you going to work on it/finish it this week?
